MaterialLib/MPL/Properties/PermeabilityOrthotropicPowerLaw.h

```diff
@@ -0,0 +1,77 @@
+/**
+ * \file
+ * \author Norbert Grunwald
+ * \date   27.06.2018
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "MaterialLib/MPL/Property.h"
+
+
+namespace ParameterLib
+{
+struct CoordinateSystem;
+}
+
+namespace MaterialPropertyLib
+{
+class Medium;
+class Phase;
+class Component;
+
+/// Orthotropic permeability model based on power law dependency on porosity.
+/// \details This property must be a solid phase property, it
+/// computes the permeability depending on the porosity. A local coordinate
+/// system can be given for orthotropy.
+template <int DisplacementDim>
+class PermeabilityOrthotropicPowerLaw final : public Property
+{
+private:
+    Phase* _phase = nullptr;
+    /// Intrinsic permeabilities, one for each spatial dimension.
+    std::array<double, DisplacementDim> const _k;
+    /// Exponents, one for each spatial dimension.
+    std::array<double, DisplacementDim> const _lambda;
+    ParameterLib::CoordinateSystem const* const _local_coordinate_system;
+
+public:
+    PermeabilityOrthotropicPowerLaw(
+        std::array<double, DisplacementDim> intrinsic_permeabilities,
+        std::array<double, DisplacementDim>
+            exponents,
+        ParameterLib::CoordinateSystem const* const local_coordinate_system);
+
+    void setScale(
+        std::variant<Medium*, Phase*, Component*> scale_pointer) override
+    {
+        if (std::holds_alternative<Phase*>(scale_pointer))
+        {
+            _phase = std::get<Phase*>(scale_pointer);
+            // TODO (naumov) Test phase is solid phase.
```

Comment:
Are there any reason why this isn't done in this PR?

Reply:
It was simpler, then I thought...